SupportYourLocals.WPF/MainWindow.xaml.cs

```diff
@@ -115,6 +124,17 @@ private void LoadAddLocalSellerFieldsAndCollections()
             }
         }
 
+        private void AddValueToNewMarketPlaceGrid()
```

Comment:
The function name is very misleading. This function adds work days to the work day combo box.